sdk/opa.go

```diff
@@ -344,6 +344,19 @@ func (opa *OPA) executeTransaction(ctx context.Context, record *server.Info, wor
 		work(s, result)
 	}
 
+	// Decision log masking only supports masking input documents that are map[string]interface{}
+	// or []interface{}. If the input was not one of those types, rebuild it from the AST.
+	switch (*record.Input).(type) {
+	case map[string]interface{}, []interface{}:
+		break
+	default:
+		asJSON, err := ast.JSON(record.InputAST)
```

Comment:
💭 I was wondering a bit about some unnecessary work here -- since the execute path needs to do something similar -- but that has already been taken into account: we're using `record.InputAST` here, not `record.Input`. :+1:

Reply:
I realized another way to avoid unnecessary work is to move this logic into the if statement that sees if the logger is registered with the manager. There is no sense in doing this unless we are going to actually log something.